rfps/rfp-consensus-visualization-tool.md

```diff
@@ -0,0 +1,123 @@
+# RFP: Consensus Visualization Tool
+
+**Name of Project:** `Consensus Visualization Tool`
+
+**Proposal Category:** `app-dev`
+
+## Project Description
+
+We are seeking proposals for an open source blockchain consensus visualization tool that demonstrates Filecoin's Expected Consensus process. There is currently a chain visualizer at http://user.kittyhawk.wtf:8000/ however it only shows the heaviest chain and does not include potential chain forks.
+
```

Comment:
Can you add something like: `We will describe a little bit more about Expected Consensus (EC) and how it works.`